doomgrader.sh

```diff
@@ -2,15 +2,40 @@
 
 set -e
 
-# doomgrader config
+# doomgrader config; EDIT HERE!
 DOOMGRADER_ROOT=~/doomgrader
 STEAM_PATH=~/.steam/steam/steamapps/common
 
 # internal paths
 DOWNLOAD_PATH=$DOOMGRADER_ROOT/files
 DEPOTDOWNLOADER_PATH=$DOOMGRADER_ROOT/depotdownloader
 
-# prompt for steam credentials
+# function for copying files with prompt
+DOOMGRADER_COPY()
+{
+# copy game files to steam dir
+echo "Copying the game from $DOWNLOAD_PATH to $STEAM_PATH/DOOMEternal/"
+echo "THIS WILL OVERWRITE YOUR EXISTING DOOM ETERNAL GAME FILES!"
+read -p "Is this correct (y/n)?: " DOOMGRADER_ANSWER
+
+if [[ $DOOMGRADER_ANSWER == 'Y' || $DOOMGRADER_ANSWER == "yes" || $DOOMGRADER_ANSWER == "YES" || $DOOMGRADER_ANSWER == "y" || $DOOMGRADER_ANSWER == "Yes" ]]
+then
+    cp $DOWNLOAD_PATH/* $STEAM_PATH/DOOMEternal/ -rfv
+else
+    echo "Copying of files stopped!"
+    echo "No files were copied or overwritten!"
+fi
+}
+
+# if the "-c" option/flag is passed, then just do the copy and finish
+if [[ $1 == "-c" ]]
+then
+DOOMGRADER_COPY
+exit 0
+fi
+
+# handles downloading depotdownloader and downloading the old version of Doom Eternal
```

Comment:
I don't think we need this comment as it's covered in the readme

Reply:
Will remove!